### PR TITLE
Update to latest TargetPlatform enum

### PR DESCRIPTION
**Pull request by vchelaru**

### CODE REVIEW

File: Documentation/mgcb.md [hunk 1]
```diff
@@ -50,13 +50,16 @@ Set the target platform for this build. It must be a member of the TargetPlatfor
 * Xbox360
 * iOS
 * Android
-* Linux
+* DesktopGL
 * MacOSX
 * WindowsStoreApp
 * NativeClient
+* PlayStationMobile
```
Comment (tomspilman) [inline Documentation/mgcb.md]: Can you actually remove `PlayStationMobile` from here... it should no longer be used and has been marked obsolete in the code base.

Comment (vchelaru) [inline Documentation/mgcb.md]: Blah github won't let me edit on the site and it's a pain to do it through sourcetree/command line. Mind accepting and I'll fix right away after with another PR?

Comment (cra0zy) [inline Documentation/mgcb.md]: You could just create a second PR, its a minor change so it will take you just a minute to set up...

**Comment by tomspilman** [comment]
No worries @vchelaru ...  I'll fix this up in my next PR.

**Pull request merged**